packages/iris-grid/src/IrisGridModel.ts

```diff
@@ -484,6 +484,13 @@ abstract class IrisGridModel<
    */
   abstract commitPending(): Promise<void>;
 
+  /**
+   * Check if viewport is still loading data
+   */
+  get hasPendingOperations(): boolean {
+    return false;
+  }
+
```

Comment:
I know I said to name it `hasPendingOperations`, but this is really viewport specific - I'd rename it `isViewportPending` or `isViewportLoading` instead.